lib/std/Thread.zig

```diff
@@ -76,6 +76,188 @@ pub fn spinLoopHint() void {
     }
 }
 
+const maxNameLen = switch (std.Target.current.os.tag) {
+    .linux => 15,
+    .windows => 31,
```

Comment:
Where does this limit come from? I've read the documentation about SetThreadDescription and it doesn't mention a 31 (or 32) as a maximum length

Reply:
It's arbitrary, I chose something which seems appropriate for a thread name.